### PR TITLE
completed deliverables except translator & comparing arrays (issues)

### PR DESCRIPTION
**Pull request by LarryMays92**
**Deliverable Submission**

Please describe your comfort and completeness levels before submitting.

Comfort Level (1-5): 2

Completeness Level (1-5):4

What did you think of this deliverable?: i need review

### CODE REVIEW

File: js/comparingarrays.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+// creates a function
+function comArray(firstArray, secondArray) {
+    if(firstArray.length === secondArray.length) {
+        for (let i = 0; i < firstArray.length; i++) {
+            if (firstArry[i]=== secondArray[i]) {
+                 if ( i === firstArray.length) {
+                    console.log("The arrays are equal")
+                }
+            } else {
+                console.log('The arrays are equal')
```
Comment (kestler01) [inline js/comparingarrays.js:14]: take a second look at our flow here with the nested conditions 
currently the only way to not be equal is with different lengths

File: js/larger.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const numbers = [2,3,4,5,6]
+const num1 = [Math.floor(Math.random()* numbers.length)]
+const num2 = [Math.floor(Math.random() * numbers.length)]
```
Comment (kestler01) [inline js/larger.js:3]: what is the above block achieving - can we make it more 'dry' ?

File: js/translator.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+//let en = " Hello World "
+//let es = ' Hola Mundo '
+//let vn = " Chao The "
+//let fr = " Bonjour Le Monde "
+
+const lang = [' Hola Mundo ', ' Chao The ', 'Bonjour Le Monde']
+console.log(Math.floor(Math.random() * lang.length))
+if(lang.length === 0) {
```
Comment (kestler01) [inline js/translator.js:8]: what is lang.length ? what should we use instead ?

**Comment by kestler01** [comment]
Great effort here- and some good creative thinking - especially love that you accounted for the '0' edge case in the plural izer. I want you to revisit the 2 questions left and resubmit after you get to a working solution. You're SO close. marking incomplete for now let us know when you open a new pull

**Comment by LarryMays92** [comment]
New pull opened

**Comment by LindZCoding** [comment]
Great work! Now marking as complete!

**Pull request closed without merging**